src/cloud-api-adaptor/entrypoint.sh

```diff
@@ -33,6 +33,7 @@ optionals+=""
 [[ "${SECURE_COMMS_PP_OUTBOUNDS}" ]] && optionals+="-secure-comms-pp-outbounds ${SECURE_COMMS_PP_OUTBOUNDS} "
 [[ "${SECURE_COMMS_KBS_ADDR}" ]] && optionals+="-secure-comms-kbs ${SECURE_COMMS_KBS_ADDR} "
 [[ "${PEERPODS_LIMIT_PER_NODE}" ]] && optionals+="-peerpods-limit-per-node ${PEERPODS_LIMIT_PER_NODE} "
+[[ "${DISABLECVM}" = "true" ]] && optionals+="-disable-cvm "
```

Comment:
You're adding a new option for docker provider, IMO it'd make sense to add it to the `install/overlays/docker/kustomization.yaml` what do you think?

Reply:
should that be `==`?